.github/workflows/tripy-l1.yml

```diff
@@ -36,3 +36,10 @@ jobs:
       run: |
         cd /tripy/
         pytest --cov=tripy/ --cov-config=.coveragerc tests/ -v -m "l1 and not manual" -n 4 --durations=15 --ignore tests/performance
+    
+    - name: notebook-test
+      run: |
+        unset PYTHONPATH
+        pip uninstall tripy -y
+        cd /tripy/
```

Comment:
We want to use the release packages here right? Not the TOT build of Tripy.

Reply:
right, they are installed in the first code block of the notebook. Here I uninstall the from source tripy so I can install later in the notebook

Reply:
ok added tripy TOT install here too